SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java

```diff
@@ -0,0 +1,62 @@
+package alexandr_samohvalov;
+
+import java.util.Scanner;
+
+public class ConditionsMain {
+    // Check if a number is positive, negative, or zero.
+    public static void maaain(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int a = scanner.nextInt();
+        if (a > 0) {
+            System.out.println("Number is positive");
+        } else if (a < 0) {
+            System.out.println("Number is negative");
+        } else {
+            System.out.println("Number is equal to zero");
+        }
+    }
+
+    //Check if a number is divisible by both 2 and 3.
+    public static void s(String[] args) {
```

Comment:
method name s is incompatible 
method name should dscribe what this method does

Reply:
same with paramters